R/utils.R

```diff
@@ -0,0 +1,44 @@
+#' helper function to return downgrade messages for lifecycle warnings
+#' @param labkey T/F whether to include temporary labkey message
+#' @noRd
+downgrade_message <- function(labkey = F) {
```

Comment:
I wonder if this amount of text can be handled in the future via a text data file. For now it is good though

Reply:
Good suggestion! Will keep as is for now.